app/src/test/java/invoice_automation/module/QuickBooksModuleTest.java

```diff
@@ -97,4 +107,28 @@ public void testGetAllCustomers_dataServiceThrows() throws Exception {
         // Run
         quickBooksModule.getAllCustomers();
     }
+
+    @Test
+    public void sendInvoice_happyPath() throws Exception {
+        // Setup
+        setupMethodTests();
+        setupInvoiceTests();
+
+        // Run
+        quickBooksModule.sendInvoice(invoice);
+
+        // Verify
+        assertEquals(EmailStatusEnum.EMAIL_SENT, invoice.getEmailStatus());
```

Comment:
We should also check that the DataService's sendEmail method got called, since all this checks right now is whether the email status was changed to sent.